File: abc/abc358/abc358_a/abc358_a.java
import java.util.*;

class Main {
  public static void main(String args[]) {
    Scanner sc = new Scanner(System.in);

    String s = sc.next();
    String t = sc.next();
    System.out.print(s.equals("AtCoder") && t.equals("Land") ? "Yes" : "No");
  }
}

File: abc/abc358/abc358_a/abc358_a.cs
using static System.Console;

public class A
{
  static void Main()
  {
    string[] a = ReadLine().Split();
    string s = a[0];
    string t = a[1];
    Write(s == "AtCoder" && t == "Land" ? "Yes" : "No");
  }
}
